docs/tech/https.md

```diff
@@ -0,0 +1,111 @@
+# HTTPS
+
+[HTTPS](https://en.wikipedia.org/wiki/HTTPS) secures information when it's
+being transmitted over the internet. Everybody should use HTTPS, but it is
+especially important that we use HTTPS because of the sensitive data we transmit.
+
+## Compatibility
+
+Some of this will be trial and error. Some settings can be tweaked on our side but
```

Comment:
Maybe worth adding a suggestion to use the SSLlabs test to check which standards are currently supported?

Reply:
Good idea 👍